control/trajectory_follower/lane_departure_checker/include/lane_departure_checker/lane_departure_checker_node.hpp

```diff
@@ -53,35 +61,31 @@ class LaneDepartureCheckerNode : public rclcpp::Node
 private:
   // Subscriber
   autoware_utils::SelfPoseListener self_pose_listener_{this};
-  rclcpp::Subscription<geometry_msgs::msg::TwistStamped>::SharedPtr sub_twist_;
-  rclcpp::Subscription<autoware_lanelet2_msgs::msg::MapBin>::SharedPtr sub_lanelet_map_bin_;
-  rclcpp::Subscription<autoware_planning_msgs::msg::Route>::SharedPtr sub_route_;
-  rclcpp::Subscription<autoware_planning_msgs::msg::Trajectory>::SharedPtr
-    sub_reference_trajectory_;
-  rclcpp::Subscription<autoware_planning_msgs::msg::Trajectory>::SharedPtr
-    sub_predicted_trajectory_;
-  rclcpp::Subscription<geometry_msgs::msg::PoseWithCovarianceStamped>::SharedPtr sub_pose_with_cov_;
+  rclcpp::Subscription<nav_msgs::msg::Odometry>::SharedPtr sub_odom_;
+  rclcpp::Subscription<HADMapBin>::SharedPtr sub_lanelet_map_bin_;
+  rclcpp::Subscription<HADMapRoute>::SharedPtr sub_route_;
+  rclcpp::Subscription<Trajectory>::SharedPtr sub_reference_trajectory_;
+  rclcpp::Subscription<Trajectory>::SharedPtr sub_predicted_trajectory_;
 
   // Data Buffer
+  nav_msgs::msg::Odometry::ConstSharedPtr current_odom_;
   geometry_msgs::msg::PoseStamped::ConstSharedPtr current_pose_;
-  geometry_msgs::msg::TwistStamped::ConstSharedPtr current_twist_;
```

Comment:
[nits] Why did you change the order (to before `current_pose_`)? The diff looks a bit strange.

Reply:
There is no reason to change the order. I fixed.
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/499/commits/4c7a50050fa8e70b19f2b11a8d486ae807df344b